packages/uploadthing/src/internal/handler.ts

```diff
@@ -315,6 +315,8 @@ export const buildRequestHandler = <TRouter extends FileRouter>(
           });
         }
 
+        console.log("Parsed config", parsedConfig);
```

Comment:
remove.

Reply:
Back on the idea of a logger, this sort of thing could be kept in if it could be flagged behind a higher log level. ed `DEBUG_LOGS=true`